client/src/api/initAxios.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+// import store from '../redux/store'
+
+export default function initAxios() {
+
+    initAxiosConfiguration();
+    initAxiosInterceptors();
+
+}
+
+function initAxiosConfiguration() {
+
+}
+
+function initAxiosInterceptors() {
+    axios.interceptors.request.use(function (request) { // .ts >> AxiosRequestConfig: request
+
+        // console.log("✉️ intercepted request")
```

Comment:
do not leave commented code when you commit

Reply:
Done
https://github.com/Levayv/ipma_v2/commit/e59e42e0f616b2de8d0c48f3f1c311eea0ab8a53